fullstop-web/fullstop-violations-rest/src/main/java/org/zalando/stups/fullstop/web/converter/ViolationEntityToDtoConverter.java

```diff
@@ -43,6 +43,15 @@ public Violation convert(ViolationEntity source) {
 
         violation.setUsername(source.getUsername());
 
+        if (source.getApplication() != null) {
+            violation.setApplicationId(source.getApplication().getName());
```

Comment:
can we rename the applicationId property to application?
right know the two setter have than different names. (setApplicationId<-> setApplicationVersion)

in the Violation class from (obviously also getter and setter):
```
    private String applicationId;

    private String applicationVersion;
```

to:
```
    private String application;

    private String applicationVersion;
```
or:
```
    private String applicationId;

    private String applicationVersionId;
```